doc/rst/high_bandwidth_endpoints.rst

```diff
@@ -0,0 +1,146 @@
+|newpage|
+
+************************************
+High bandwidth isochronous endpoints
+************************************
+
+``lib_xud`` provides support for high bandwidth isochronous endpoints on high-speed USB devices, as described in the
+Section 5.9.2 of the `USB 2.0 specification <https://usb.org/document-library/usb-20-specification>`_.
+This enables higher data throughput by allowing multiple transactions per microframe.
+
+The current implementation in ``lib_xud`` has the following limitations:
```

Comment:
I would remove "current" - we don't want to put ourselves on the hook for removing these.

Reply:
definitely not. done!